packages/transaction-pay-controller/CHANGELOG.md

```diff
@@ -7,10 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.6.0]
+
+### Added
+
+- feat: add override functionality to remote feature flags ([#7271](https://github.com/MetaMask/core/pull/7271))
```

Comment:
I don't think this is needed as this just changes to tests for this package, is this true?